src/Data/Container/Relation/Binary/Equality/Setoid.agda

```diff
@@ -0,0 +1,54 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Equality over container extensions parametrised by some setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary using (Setoid)
+
+module Data.Container.Relation.Binary.Equality.Setoid {c e} (S : Setoid c e) where
+
+open import Level using (_⊔_; suc)
+open import Relation.Binary
+
+open import Data.Container.Core
+open import Data.Container.Relation.Binary.Pointwise
+import Data.Container.Relation.Binary.Pointwise.Properties as Pw
+
+private
+  module S = Setoid S
+open S using (_≈_) renaming (Carrier to X)
+
+------------------------------------------------------------------------
+-- Definition of equality
+
+module _ {s p} (C : Container s p) where
+
+  Eq : Rel (⟦ C ⟧ X) (e ⊔ s ⊔ p)
```

Comment:
Is it worth it changing this to `_≋_`? Seems like a nice convention to have for equality over data that contains multiple elements (e.g. List, Vec, Container) etc.

Reply:
I don't think `C` can be inferred. :/

Reply:
Haha, my comment about not understanding about this part of the library was spot on!